<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignVerifyDlg</class>
 <widget class="QDialog" name="SignVerifyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signature / Verify</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="0" colspan="6">
    <widget class="QLabel" name="mStatusLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="mCertPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="mCertBtn">
       <property name="text">
        <string>Certificate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Property and Method</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QPlainTextEdit" name="mInputText"/>
   </item>
   <item row="13" column="0" colspan="6">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mInputStringRadio">
       <property name="text">
        <string>String</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mInputHexRadio">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mInputBase64Radio">
       <property name="text">
        <string>Base64</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Len</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mInputLenText">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="mAlgTypeCombo"/>
     </item>
     <item>
      <widget class="QComboBox" name="mHashTypeCombo"/>
     </item>
     <item>
      <widget class="QComboBox" name="mVersionCombo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mCheckKeyPairBtn">
       <property name="text">
        <string>Check KeyPair</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mMethodCombo"/>
     </item>
    </layout>
   </item>
   <item row="14" column="0" colspan="6">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="4">
      <widget class="QPushButton" name="mRunBtn">
       <property name="text">
        <string>Sign</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mUpdateBtn">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="mCloseBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="mInitBtn">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mFinalBtn">
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="6">
    <widget class="QPlainTextEdit" name="mOutputText"/>
   </item>
   <item row="5" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="mPriKeyPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="mPriKeyBtn">
       <property name="text">
        <string>Private Key</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="3">
      <widget class="QCheckBox" name="mPubKeyVerifyCheck">
       <property name="text">
        <string>Verify with public key</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mPriKeyAndCertLabel">
       <property name="text">
        <string>Private key and Certificate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="mAutoCertPubKeyCheck">
       <property name="text">
        <string>Cert or PubKey Auto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="6">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mOutputLenText">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output Data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Len</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mInputStringRadio</tabstop>
  <tabstop>mInputHexRadio</tabstop>
  <tabstop>mInputBase64Radio</tabstop>
  <tabstop>mInputLenText</tabstop>
  <tabstop>mInputText</tabstop>
  <tabstop>mPubKeyVerifyCheck</tabstop>
  <tabstop>mPriKeyPath</tabstop>
  <tabstop>mPriKeyBtn</tabstop>
  <tabstop>mCertPath</tabstop>
  <tabstop>mCertBtn</tabstop>
  <tabstop>mAlgTypeCombo</tabstop>
  <tabstop>mHashTypeCombo</tabstop>
  <tabstop>mVersionCombo</tabstop>
  <tabstop>mCheckKeyPairBtn</tabstop>
  <tabstop>mMethodCombo</tabstop>
  <tabstop>mOutputLenText</tabstop>
  <tabstop>mOutputText</tabstop>
  <tabstop>mInitBtn</tabstop>
  <tabstop>mUpdateBtn</tabstop>
  <tabstop>mFinalBtn</tabstop>
  <tabstop>mRunBtn</tabstop>
  <tabstop>mCloseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
